Python/chapter01/1.1 - Is Unique/Q1.1.py

```diff
@@ -0,0 +1,27 @@
+
+# Note that this solution assumes a unique string
+# implies a lowercase letter and the uppercase of
+# that letter is not considered duplicates
+# example: the string "Hather" has an uppercase "H"
+#          and lowercase "h" yet the string has no duplicate
+#          "h" or "H"
+
+
+def uniqueChar(passedStr):
+
+        # Will use the built-in function set() which
```

Comment:
clarification, [`set`](https://docs.python.org/3/library/stdtypes.html#set) is a class/type, not a function. you're just invoking the class' constructor here.